docker/ftest-jpa/src/main/java/org/arquillian/cube/persistence/EmployeeRepository.java

```diff
@@ -0,0 +1,33 @@
+package org.arquillian.cube.persistence;
+
+import java.util.List;
+import javax.annotation.sql.DataSourceDefinition;
+import javax.ejb.Singleton;
+import javax.ejb.Startup;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.Query;
+
+@DataSourceDefinition(name = "java:app/datasources/postgresql_ds",
+    className = "org.postgresql.ds.PGSimpleDataSource",
+    //url = "jdbc:postgresql://postgres:5432/", // when running mvn clean install -Pwildfly-12-remote
+    url = "jdbc:postgresql://localhost:5433/",
```

Comment:
@bartoszmajsak I couldn't find a way how can you tweak this with maven profile, as we'll be having different urls in both profiles(`managed` and `remote`)

In managed mode wildfly is running on localhost which can talk to postgres exposed on `5433`, whereas in remote mode, wildfly can't talk on localhost as it's running in container which is linked to `postgres` so we have to use host as `postgres` when running it in remote mode.

While running on travis it seems that `5432` post is already exposed by some service due to which, I am exposing `5433` port on localhost.

Thoughts?

Reply:
The problem is that `@DataSourceDefinition` is not system env aware. Hummm no idea now, I think that you could go tight to wildfly conf files https://developer.jboss.org/thread/267548

Reply:
This will make it WF specific - are we fine with that?

Reply:
I know but I do not see any other option JavaEE spec is restricted in this way.

Reply:
So maybe we run those tests only against containerized setup using our conditional annotations for tests?

Reply:
Which I believe we do right now regardless

Reply:
@dipak-pawar would container-only approach make us confident enough to release new cube with this fix?

Reply:
@bartoszmajsak, Yes, this is just example. I believe we are goog to release a new cube with this fix.